e2e/app/container.ts

```diff
@@ -67,14 +67,14 @@ export default class Container {
       )([
         {
           shouldWait: waitForNav,
-          waitFn: () => {
-            this.page.waitForNavigation({ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout });
+          waitFn: async () => {
+            await this.page.waitForNavigation({ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout });
```

Comment:
Revert back to code before commit [4993](https://github.com/all-of-us/workbench/pull/4993/files).